pallets/ethereum-events/src/benchmarking.rs

```diff
@@ -144,8 +144,8 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::RuntimeEvent) {
 
 benchmarks! {
     add_validator_log {
-        let u in 1 .. MAX_NUMBER_OF_UNCHECKED_EVENTS;
-        let e in 1 .. MAX_NUMBER_OF_EVENTS_PENDING_CHALLENGES;
+        let u = MAX_NUMBER_OF_UNCHECKED_EVENTS-2;
```

Comment:
Instead of constantly having these limits written as `MAX... - n`, perhaps we could have a single constant with that value that is used everywhere else.
(don't do this now, add a Jira task to come back to this and do it later, I don't think this is important enough to delay this PR any longer)